lib/rspec/core/formatters/c_formatter.rb

```diff
@@ -0,0 +1,18 @@
+RSpec::Support.require_rspec_core "formatters/base_formatter"
+
+module RSpec
+  module Core
+    module Formatters
+      # @private
+      class CFormatter < BaseFormatter
+        Formatters.register self, :example_failed, :dump_profile
+
+        def example_failed(failure)
+          output.puts "#{failure.example.location}:#{failure.example.description}"
+        end
+
+        def dump_profile(_profile); end
```

Comment:
You don't need to do this if you don't register `dump_profile`

Reply:
I guess I put this because I was testing [a project where the suggested `config.profile_examples = 10` setting was set](https://github.com/smortex/certificate-checker/blob/8999ef2a28e24b30275aef9827d6fe12c7f04c55/spec/spec_helper.rb#L92), and this output is not processable.

While I could disable this for the project, I though it would be wiser to silently discard the information when running with this formatter and keep the profiler reports when running with another formatter.

Is there a more elegant / less surprising way to say "do not do profiling when using this formatter"?